ignek_intranet/modules/common/src/main/java/com/ignek/intranet/common/constants/IntranetConstants.java

```diff
@@ -0,0 +1,102 @@
+ */
+public class IntranetConstants {
+
+	public static final String EMP_ID = "empId";
+
+	public static final String EMPLOYEE_LIST = "employeeList";
+
+	public static final String FIRST_NAME = "firstName";
+
+	public static final String LAST_NAME = "lastName";
+
+	public static final String EMAIL_ADDRESS = "emailAddress";
+
+	public static final String PHONE_NUMBER = "phoneNumber";
+
+	public static final String ADDRESS_LINE_1 = "addressLine1";
+
+	public static final String ADDRESS_LINE_2 = "addressLine2";
+
+	public static final String CITY = "city";
+
+	public static final String ZIPCODE = "zipCode";
+
+	public static final String DESIGNATION = "designation";
+
+	public static final String BIRTHDAY_DAY = "01";
+
+	public static final String BIRTHDAY_MONTH = "01";
+
+	public static final String BIRTHDAY_YEAR = "1990";
+
+	public static final String EMPLOYEE_ROLE = "Employee";
+
+	public static final String REPORT_FIRST_NAME = "First Name";
+
+	public static final String REPORT_LAST_NAME = "Last Name";
+
+	public static final String REPORT_EMAIL_ADDRESS = "Email Address";
+
+	public static final String REPORT_PHONE_NUMBER = "Phone Number";
+
+	public static final String REPORT_ADDRESS_LINE_1 = "ADDRESS LINE 1";
+
+	public static final String REPORT_ADDRESS_LINE_2 = "ADDRESS LINE 2";
+
+	public static final String REPORT_CITY = "City";
+
+	public static final String REPORT_ZIPCODE = "Zipcode";
+
+	public static final String REPORT_DESIGNATION = "Designation";
+
+	public static final String USER_ID = "userId";
+
+	public static final String ACTIVITY_EVENT = "C_ActivityEvent";
+
+	public static final String ACTIVITY_CREATED_DATE = "createdDate";
+
+	public static final String ACTIVITY_UPDATED_DATE = "updatedDate";
+
+	public static final String ACTIVITY_USER_ID = "activityUserId";
+
+	public static final String ACTIVITY_IP_ADDRESS = "iPAddress";
+
+	public static final String ACTIVITY_TYPE = "activityType";
+
+	public static final String APPLICATION_PDF = "application/pdf";
+
+	public static final String ATTACHEMENT_FILENAME = "attachement;filename=";
+
+	public static final String PDF_EXTENTION = ".pdf";
+
+	public static final String HR_ROLE = "HR";
+
+	public static final String CONSTRAINT_VIOLATION_EXCEPTION = "ConstraintViolationException";
+
+	public static final String EMPLOYEE_CREATED_STATUS_MESSAGE = "employee-created";
+
+	public static final String EMPLOYEE_NOT_CREATED_STATUS_MESSAGE = "employee-not-created";
+
+	public static final String EMPLOYEE_UPDATED_STATUS_MESSAGE = "employee-updated";
+
+	public static final String EMPLOYEE_NOT_UPDATED_STATUS_MESSAGE = "employee-not-updated";
+
+	public static final String EMPLOYEE_DELETED_STATUS_MESSAGE = "employee-deleted";
+
+	public static final String EMPLOYEE_NOT_DELETED_STATUS_MESSAGE = "employee-not-deleted";
+
+	public static final String ERROR_FOR_EXISTING_EMAIL_STATUS_MESSAGE = "error-for-existing-email";
+	
+	public static final String WEB_CONTENT_TITLE = "Text40079433";
```

Comment:
Why this hardcoded id? will it work if I create a new structure?

Change it for below three as well.